pinot-core/src/test/java/org/apache/pinot/core/data/function/DateTimeFunctionEvaluatorTest.java

```diff
@@ -52,12 +54,14 @@ public void testDateTimeTransformFunctions(String transformFunction, List<String
     // toEpochSeconds w/ rounding
     GenericRow row1_1 = new GenericRow();
     row1_1.putValue("timestamp", 1578685189000L);
-    inputs.add(new Object[]{"toEpochSecondsRounded(timestamp, 10)", Lists.newArrayList("timestamp"), row1_1, 1578685180L});
+    inputs.add(
+        new Object[]{"toEpochSecondsRounded(timestamp, 10)", Lists.newArrayList("timestamp"), row1_1, 1578685180L});
```

Comment:
Ended up getting a good number of formatting differences when I auto-formatted this file. Is my setup wrong or do y'all get these changes when formatting too?

Reply:
If you've taken the pinot style as mentioned in the Pinot docs, then you're fine.